src/crimson/os/seastore/random_block_manager/nvme_block_device.cc

```diff
@@ -20,7 +20,7 @@ namespace {
 
 namespace crimson::os::seastore::nvme_device {
```

Comment:
Namespace nvme_device looks inappropriate.

Probably rename to random_block_device or block_device ?